### PR TITLE
Add my wayland-related bindings

### PR DESCRIPTION
**Pull request by ifreund**

### CODE REVIEW

File: packages/zig-wayland.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "author": "Isaac Freund",
+  "tags": [
+    "binding"
+  ],
+  "git": "https://github.com/ifreund/zig-wayland",
+  "root_file": "",
```
Comment (ifreund) [inline packages/zig-wayland.json:7]: Not sure what to put here as the root file is generated using a provided build step. See the README for details: https://github.com/ifreund/zig-wayland

Comment (MasterQ32) [reply packages/zig-wayland.json:7]: Hmm, good question… Maybe we should allow passing `null` here to allow to signal a more complicated setup. Same for @Snektron​s vulkan wrapper

Comment (Luukdegram) [reply packages/zig-wayland.json:7]: @ifreund Updated the repo to allow `null` to be passed into `root_file` (#19).

Comment (ifreund) [reply packages/zig-wayland.json]: @Luukdegram Thanks! I've updated the commit.

**Comment by Luukdegram** [comment]
Thanks!

**Pull request merged**